.github/workflows/ms.automation.automationaccounts.yml

```diff
@@ -4,17 +4,25 @@ on:
   workflow_dispatch:
     inputs:
       removeDeployment:
+        type: boolean
         description: 'Remove deployed module'
         required: false
         default: 'true'
       versioningOption:
-        description: 'The mode to handle the version increments [major|minor|patch]'
+        type: choice
+        description: 'The mode to handle the version increments [major|minor|patch|custom]'
         required: false
         default: 'patch'
+        options:
+          - major
+          - minor
+          - patch
+          - custom
```

Comment:
Should we do this only here or rather tackle this in a PR for all?

Reply:
The new inputs feature that is

Reply:
I'd suggest one search and replace and be gone with it. We should definitely get them in asap

Reply:
I'll do that shortly after this PR is merged. I'd like to test the push to main